external-crates/move/crates/move-vm-runtime/src/unit_tests/basic_block_tests.rs

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) The Move Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{
+    dev_utils::{
+        compilation_utils::compile_packages_in_file, in_memory_test_adapter::InMemoryTestAdapter,
+        vm_test_adapter::VMTestAdapter,
+    },
+    jit::optimization,
+};
+use move_core_types::account_address::AccountAddress;
+
+#[test]
+fn test_basic_blocks_0() {
+    let package_address = AccountAddress::from_hex_literal("0x1").unwrap();
+    let mut adapter = InMemoryTestAdapter::new();
+    for pkg in compile_packages_in_file("basic_blocks.move", &[]) {
+        adapter.insert_package_into_storage(pkg);
+    }
+    let serialized_package = adapter.get_package_from_store(&package_address).unwrap();
+    let (verif_pkg, _) = adapter
+        .verify_package(package_address, serialized_package)
+        .unwrap();
+    let pkg = optimization::to_optimized_form(verif_pkg);
+    println!("Blocks\n---------------------------\n{:#?}", pkg.modules);
```

Comment:
I can see why you have this ;) But keeping it as a test doesn't make sense since it isn't testing anything. I would either remove, write some assertions over the optimized form, or maybe mark with a TODO to remove later on.

Reply:
The fact that this optimization passes is quasi-sufficient as a "test", but I will write a TODO to add more.